src/main/java/com/marginallyclever/makelangelo/plotter/plotterControls/MarlinPlotterInterface.java

```diff
@@ -135,7 +135,7 @@ protected void onHearAcceleration(String message) {
 			logger.debug("MarlinPlotterInterface found acceleration {}", v);
 			myPlotter.getSettings().setAcceleration(v);
 		} catch (Exception e) {
-			logger.error("M201 error: {}", message, e);
+			logger.warn("M201 error: {}", message, e);
```

Comment:
why the downgrade to warning?  it literally says "error" in the output.

Reply:
yep, but this is not fatal for the application so warning is good enough. If it was an error, there will be a message to the user or something else in the catch. Here nothing is done except a log.